lsp-semantic-tokens.el

```diff
@@ -61,6 +61,15 @@ preferred over both full and ranged token requests."
   :group 'lsp-semantic-tokens
   :type 'boolean)
 
+(defcustom lsp-semantic-tokens-honor-refresh-requests nil
```

Comment:
This frequency should be different form language server to language server so maybe it's a good idea to have it disabled or even in the future we allow/enable for servers we know do that "right"